paasta_tools/cli/utils.py

```diff
@@ -596,6 +596,29 @@ def execute_paasta_metastatus_on_remote_master(
     )
 
 
+def get_service_autoscale_pause_time(cluster):
```

Comment:
I think ``paasta_tools/autoscaling/autoscaling_service_lib.py`` would be a better home for these functions

Reply:
It would take me a decent amount of time to make this change and I'm not 100% convinced the autoscaling service lib is the place right place for them. I would rather keep these functions in this file since they're only being used by the command line tool, and I'm not sure where else we would need to use them again anyway. Currently I believe the autoscaling service lib file only contains functions used by the autoscaler which these are not.

Reply:
Or how about just pause_service_autoscaler.py ?